### PR TITLE
Fixing Findbug issues

### PR DESCRIPTION
**Pull request by Drifftr**

### CODE REVIEW

File: src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java [hunk 1]
```diff
@@ -20,87 +20,35 @@
...
-			fileArray[i] = allFilesPresentInFolder.get(i);
-			log.info("FileArray name : " + allFilesPresentInFolder.get(i).getName());
-		}
-		List<File> validFileList = getFileList(fileArray);
-		if (validFileList.size() == 0) {
-			throw new Exception("The selected location " + location.getName() + "(" + location.toString()
-					+ ") does not contain any human task files.");
-		}
 		File targetFolder;
 		targetFolder = new File(location.getPath(), "target");
```
Comment (vinok88) [inline src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java]: make the string "target" a contant

File: src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java [hunk 2]
```diff
@@ -141,28 +89,29 @@ static private void addToZip(String path, String srcFile, ZipOutputStream zip) {
...
 		File folder = new File(srcFolder);
-		String fileListe[] = folder.list();
+		String fileListArray[] = folder.list();
 		int i = 0;
-		if (fileListe != null) {
-			while (i < fileListe.length) {
-				addToZip("", srcFolder + File.separator + fileListe[i], zip);
+		if (fileListArray != null) {
+			while (i < fileListArray.length) {
+				addToZip("", srcFolder + File.separator + fileListArray[i], zip);
```
Comment (vinok88) [inline src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java]: Better to have a constant for empty ("") string.

File: pom.xml [hunk 3]
```diff
@@ -109,4 +108,13 @@
 		<connection>scm:git:https://github.com/wso2/maven-tooling-bps.git</connection>
 		<tag>HEAD</tag>
 	</scm>
+	<properties>
+	<maven.plugin.plugin.version>3.4</maven.plugin.plugin.version>
```
Comment (hasithaa) [inline pom.xml]: Formatting issues.

File: src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java [hunk 4]
```diff
@@ -141,28 +89,29 @@ static private void addToZip(String path, String srcFile, ZipOutputStream zip) {
...
+		String fileListArray[] = folder.list();
 		int i = 0;
-		while (i < fileListe.length) {
-			String newPath = folder.getName();
-			if (!path.equalsIgnoreCase("")) {
-				newPath = path + File.separator + newPath;
+		if (fileListArray != null) {
+			while (i < fileListArray.length) {
+				String newPath = folder.getName();
+				if (!path.equalsIgnoreCase("")) {
```
Comment (hasithaa) [inline src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java]: Move "" to a Constant.

File: src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java (hunk 1 above)
Comment (Drifftr) [inline src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java]: Fixed in https://github.com/wso2/maven-tooling-bps/pull/7/commits/43199d0a0b82d1b130de3da85e294ab25d422fd2

File: src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java (hunk 2 above)
Comment (Drifftr) [inline src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java]: Fixed in https://github.com/wso2/maven-tooling-bps/pull/7/commits/43199d0a0b82d1b130de3da85e294ab25d422fd2

File: src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java (hunk 4 above)
Comment (Drifftr) [inline src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java]: Fixed in https://github.com/wso2/maven-tooling-bps/pull/7/commits/43199d0a0b82d1b130de3da85e294ab25d422fd2

File: pom.xml (hunk 3 above)
Comment (Drifftr) [inline pom.xml]: Fixed in https://github.com/wso2/maven-tooling-bps/pull/7/commits/43199d0a0b82d1b130de3da85e294ab25d422fd2

File: src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java [hunk 5]
```diff
@@ -20,87 +20,35 @@
...
-		List<File> validFileList = getFileList(fileArray);
-		if (validFileList.size() == 0) {
-			throw new Exception("The selected location " + location.getName() + "(" + location.toString()
-					+ ") does not contain any human task files.");
-		}
 		File targetFolder;
 		targetFolder = new File(location.getPath(), "target");
 		File humantaskDataFolder = new File(targetFolder, "ht-tmp");
-		humantaskDataFolder.mkdirs();
+		if(!humantaskDataFolder.mkdirs()){
```
Comment (darshanasbg) [inline src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java]: Fix formatting issues

Comment (Drifftr) [inline src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java]: Thanks. The formatting issues were fixed in later commits.

Comment (darshanasbg) [inline src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java]: Do you have a PR for that? I still can see those issue in master[1].

[1] https://github.com/wso2/maven-tooling-bps/blob/master/src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java#L48

File: src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java (hunk 4 above)
Comment (prabathabey) [inline src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java]: What should have ideally been done here is using String.isEmpty().

Comment (Drifftr) [inline src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java]: Thanks. This has been replaced later with StringUtils.isBlank() in https://github.com/wso2/maven-tooling-bps/pull/8/commits/dd4661f06183e7ddbcb97a4898987ffa46f3a115

File: src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java (hunk 5 above)
Comment (Drifftr) [inline src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java]: Thanks for pointing out. It seems I missed this one. Fixed these in https://github.com/wso2/maven-tooling-bps/pull/10

**Pull request merged**